structuralalignment/superposition/theseus.py

```diff
@@ -1 +1,114 @@
-#here is the theseus code
+import atomium
+import subprocess
+
+# usage: Theseus(["3dk3"])
+class Theseus:
+    def __init__(self, pdbs):
```

Comment:
Aligner classes should follow a factory pattern. We preconfigure the aligner, but the actual molecules are only passed to the executor (`Theseus.align(...)` in this case). The arguments to `__init__` should only be options for the program, but not the input structures themselves.